bootloaders/riotboot/Makefile

```diff
@@ -16,6 +16,7 @@ CFLAGS += -DRIOTBOOT
 CFLAGS += -DNDEBUG -DLOG_LEVEL=LOG_NONE
 DISABLE_MODULE += core_init core_msg core_panic
 DISABLE_MODULE += auto_init
+DISABLE_MODULE += samd21_bootloader_default_module
```

Comment:
The line below could be removed as well: no other boards are setting stdio_cdc_acm as default module.

Reply:
Hmm yest, but There is PR that wants to add it #13649, so to me it makes sense to keep it.

Reply:
I'm not sure. The approach in #13649 should be done differently: define auto_init_usbus as default module and if it is used then use stdio_cdc_acm module (using USEMODULE).
And force the `stdio_null` module with riotboot ?

Reply:
Forcing `stdio_null` could make sense

Reply:
> I'm not sure. The approach in #13649 should be done differently: define auto_init_usbus as default module and if it is used then use stdio_cdc_acm module (using USEMODULE).

I don't agree with this. But anyway maybe leave that for whenever `auto_init_usbus` is handled better?

Reply:
> Forcing stdio_null could make sense

And it seems to work :)

Reply:
> But anyway maybe leave that for whenever auto_init_usbus is handled better?

We can leave it for later, but #13649 and #13276 require something and use 2 different approaches.